src/modules/account/components/AccountRow/AccountRow.js

```diff
@@ -18,9 +20,23 @@ function AccountRow({ account, onSelect, onRemove, truncate }) {
     >
       <WalletVisual address={address} size={40} />
       <div className={styles.addressWrapper}>
-        <b className={`${styles.addressValue}`}>
-          <span>{name}</span> {isHW && <Icon name="hardwareWalletIcon" />}
-        </b>
+        <div className={styles.header}>
+          <b className={`${styles.addressValue}`}>
+            <span>{name}</span> {isHW && <Icon name="hardwareWalletIcon" />}
+          </b>
+          {!(isHW && isNew) ? null : (
+            <div>
+              <span className={styles.new}>{t('New')}</span>
+              <Tooltip size="m" tooltipClassName={`${styles.tooltipContainer}`} position="left">
+                <p>
+                  {t(
+                    'Transfer tokens to the uninitialized new account to utilize all the wallet features. At any given time, only one uninitialized account can be created.'
+                  )}
+                </p>
+              </Tooltip>
+            </div>
+          )}
```

Comment:
This looks confusing I guess this also does the work and makes it readable

```suggestion
          {isHW && isNew && (
            <div>
              <span className={styles.new}>{t('New')}</span>
              <Tooltip size="m" tooltipClassName={`${styles.tooltipContainer}`} position="left">
                <p>
                  {t(
                    'Transfer tokens to the uninitialized new account to utilize all the wallet features. At any given time, only one uninitialized account can be created.'
                  )}
                </p>
              </Tooltip>
            </div>
          )}
```

Reply:
Yea, this is what i also commented on here https://github.com/LiskHQ/lisk-desktop/pull/4834#discussion_r1119825387. Thought it was fixed sine my comment was closed